### DISCUSSION TITLE
Add missing "Load" and "Save as" options next to the pre-existing Save button for configuration swapping

### DISCUSSION THREAD

**Discussion by ojsl1** [benjamimgois/goverlay#67] [Ideas]
There are times when I want to use a customized minimal configuration during casual gameplay and other times when a game is misbehaving and I need a detailed customized configuration to monitor what's going wrong. Moving my custom MangoHud.conf files around is a huge hassle in both file browser and a terminal.

PS: No, I don't want to create a shell aliases for swapping the configurations as Goverlay is supposed to be: "the project to help manage Linux overlays."

PSS: The "Quick Layouts": minimal, compact, complete and graph are not what I'm not after, as 1) don't want to use pre-made layouts and 2) don't want to edit the pre-made quick layouts (if that is even possible (which it should be(offtopic))).

Yay or nay?

**Comment by benjamimgois** [comment]
Hi @ojsl1 , thanks for all your feedback. The way Goverlay was coded, it was very dificult to implement such feature. But i'm in the middle of rewrite a bunch of code that will allow this to work in the future. I can't promiss this for the next release, but will be add to the todo list.
